pkg/config/config.go

```diff
@@ -27,18 +30,21 @@ func New() Config {
 		PostgresContainerName:   getEnv("POSTGRES_CONTAINER_NAME", "postgresql"),
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "ups-oauth-proxy"),
 
-		UPSImageStreamName:        getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
-		UPSImageStreamTag:         getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+		UPSImageStreamName: getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
+		UPSImageStreamTag:  getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+
 		OauthProxyImageStreamName: getEnv("OAUTH_PROXY_IMAGE_STREAM_NAME", "ups-oauth-proxy-imagestream"),
 		OauthProxyImageStreamTag:  getEnv("OAUTH_PROXY_IMAGE_STREAM_TAG", "latest"),
 
 		PostgresImageStreamNamespace: getEnv("POSTGRES_IMAGE_STREAM_NAMESPACE", "openshift"),
 		PostgresImageStreamName:      getEnv("POSTGRES_IMAGE_STREAM_NAME", "postgresql"),
-		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "latest"),
+		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "9.6"),
```

Comment:
The `latest` ImageStreamTag in that ImageStream provides PostgreSQL 10. Maybe that's fine, but we've only really used 9.6 up to now. @secondsun WDYT?

Reply:
It won't be a huge pain in the ass to update, everyone is good about being forward/backwards compatible, and we use Hibernate to handle SQL so in theory there shouldn't be any breaks.

Reply:
Ok cool, I'll change this to `10` for now then (currently equivalent to latest, but sounds way safer since who knows what an `11` would do).

Reply:
ups is using 9.6 psql isn't it?

Reply:
I would prefer the image tag as well. It can break with an upper version.

Reply:
@grdryn in all my tests t worked. If the ENV VAR of the encryption secret is == "" it worked. Not required the if at all.

Reply:
Going to change this to 10 for now as discussed in hangouts.

@camilamacedo86 your most recent comment here doesn't look like it's related to this line. Is it referring to some other part of the change?

Reply:
It is true. Sorry. It was over the if to do the args which are passed to the container in the cronJob.